content/zh/docs/tutorials/security/seccomp.md

```diff
@@ -0,0 +1,787 @@
+---
+reviewers:
+- hasheddan
+- pjbgf
+- saschagrunert
+title: 使用 seccomp 限制容器的系统调用
```

Comment:
How about removing "reviewers" lines?
It seems that they are EN reviewers. 😊

```suggestion
---
title: 使用 seccomp 限制容器的系统调用
```

(For anyone's information:
EN file: https://github.com/kubernetes/website/blob/main/content/en/docs/tutorials/security/seccomp.md?plain=1 )